astericks/astericks.c++

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+
+using namespace std;
+
+int main(){
+
+    int rows{};
+    int cols{};
+    cout<<"Number of rows: ";
+    cin>> rows;
+    cout<<"Number of cols: ";
+    cin>>cols;
+
+    for (int i{1}; i <= rows ; i++){

```

Comment:
any particular reason for initializing variables like this?

Reply:
yes, I'm learning C++ as I code the assignments.
one of the things I learned is that I should prefer "List initialization" over other methods to initialize my variable. 
It is also recommended by Bjarne Stroustrup.
 https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es20-always-initialize-an-object